### PR TITLE
WIP for BXMSDOC-4079-master: Modified the Entando App Builder doc content.

### PR DESCRIPTION
**Pull request by kaldesai**
Epic: https://issues.jboss.org/browse/BXMSDOC-3999

JIRA: https://issues.jboss.org/browse/BXMSDOC-4079

Rendered output link: 
[Integrating Entando-RHPAM-7.4](http://file.pnq.redhat.com/~kaldesai/BXMSDOC-4079-Entando-RHPAM-7.4/)

### CODE REVIEW

File: assemblies/assembly_integrating-entando/main.adoc [hunk 1]
```diff
@@ -1,34 +1,27 @@
...
+= Integrating {PRODUCT} assets in Entando App Builder
 // Context attribute is assembly specific and enables module reuse between assemblies.
 :doctype: book
 :imagesdir: _images
 :context: integrating-with-entando
 include::_artifacts/author-group.adoc[]
 
 // Purpose statement for the assembly
-As a business processes developer, you can integrate the business assets you have created in {PRODUCT} with Entando App Builder 5.0.3. Entando provides an open-source, low-code Digital Experience Platform (DXP) for building modern web, mobile, and IoT applications. With Entando, you can create a custom user experience around your {PRODUCT} business assets and expand your application development strategy.
+As a business processes developer, you can integrate the business assets you have created in {PRODUCT} with Entando App Builder. The App Builder provides an open-source, low-code Digital Experience Platform (DXP) for building modern web, mobile, and IoT applications. With the App Builder, you can create a custom user experience around your {PRODUCT} business assets and expand your application development strategy.
```
Comment (sterobin) [inline assemblies/assembly_integrating-entando/main.adoc]: "The App Builder"  >  "App Builder" No "the". So "App Builder provides ...", "With App Builder...", etc. Search all instances and update.

File: doc-content/enterprise-only/integration/entando-con.adoc [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 = Entando Digital Experience Platform (DXP) and App Builder
 
-Entando Digital Experience Platform (DXP) is a low-code, component-based platform for building modern web, mobile, and IoT applications. The Entando App Builder is the application that provides features to help you configure and customize, with or without programming skills, any kind of modern web application built with the Entando Platform. The Entando App Builder is a web-based administration console that you can use to build and configure your web application, manage your page data and activities, and integrate with other application tools, such as {PRODUCT}.
+Entando Digital Experience Platform (DXP) is a low-code, component-based platform for building modern web, mobile, and IoT applications. The App Builder is an application that provides features to help you configure and customize, with or without programming skills, any kind of modern web application built with the App Builder Platform. The App Builder is a web-based administration console that you can use to build and configure your web application, manage your page data and activities, and integrate with other application tools, such as {PRODUCT}.
```
Comment (sterobin) [inline doc-content/enterprise-only/integration/entando-con.adoc]: "the App Builder Platform"  Make sure to remove "Platform" here too, so it's "built with App Builder", with the change you make per my previous comment.

File: doc-content/enterprise-only/integration/entando-integrating-proc.adoc [hunk 3]
```diff
@@ -1,34 +1,11 @@
 [id='entando-integrating-proc_{context}']
 
-= Configuring {KIE_SERVERS} in Entando App Builder
+= Configuring {PRODUCT} with Entando App Builder
```
Comment (sterobin) [inline doc-content/enterprise-only/integration/entando-integrating-proc.adoc]: Since this is no longer a procedure, I'd call it "{PRODUCT} configuration with Entando App Builder".

You'll need to also change the anchor ID and file name to say "-con" instead of "-proc" BUT, first search the entire kie-docs project for both current names to find if and where they're currently used and update the refs and includes to the new name, with "-con". Otherwise it will break. This is like what we discussed the other day in our single-source convo.

File: doc-content/enterprise-only/integration/entando-integrating-proc.adoc [hunk 4]
```diff
@@ -1,34 +1,11 @@
...
 
-In order to integrate your {PRODUCT} data in Entando, you must first configure one or more {KIE_SERVERS} in Entando App Builder. The configured {KIE_SERVERS} must be the servers to which you deployed the projects that you created in {PRODUCT}.
+In order to integrate your {PRODUCT} data in App Builder, you must first configure one or more {KIE_SERVERS} in App Builder. The configured {KIE_SERVERS} must be the servers to which you deployed the projects that you created in {PRODUCT}.
 
-.Prerequisites
-* {PRODUCT} {PRODUCT_VERSION} is installed. For installation options, see {URL_PLANNING_INSTALL}[_{PLANNING_INSTALL}_].
-* One or more {KIE_SERVERS} are running and contain deployed project data. For instructions, see {URL_PACKAGING_DEPLOYING_PROJECT}[_{PACKAGING_DEPLOYING_PROJECT}_].
-* Entando App Builder 5.0.3 is installed. For instructions, see link:http://docs.entando.com/[_Getting Started with Entando App Builder_].
-* You are logged in to the Entando Admin Console.
+You can use the following links to configure Red Hat Process Automation Manager with App Builder:
```
Comment (sterobin) [inline doc-content/enterprise-only/integration/entando-integrating-proc.adoc]: I'd reword to "For instructions on getting started with App Builder and for configuring {PRODUCT} with App Builder, see the following links:"

Comment (kaldesai) [reply doc-content/enterprise-only/integration/entando-integrating-proc.adoc]: Done

File: doc-content/enterprise-only/integration/entando-con.adoc (hunk 2 above)
Comment (kaldesai) [reply doc-content/enterprise-only/integration/entando-con.adoc]: Done

File: assemblies/assembly_integrating-entando/main.adoc (hunk 1 above)
Comment (kaldesai) [reply assemblies/assembly_integrating-entando/main.adoc]: Done

File: doc-content/enterprise-only/integration/entando-integrating-proc.adoc (hunk 3 above)
Comment (kaldesai) [reply doc-content/enterprise-only/integration/entando-integrating-proc.adoc]: Done
